functions/utils/getApolloClient.ts

```diff
@@ -0,0 +1,21 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+const GRAPHQL_ENDPOINT = 'https://api.uniswap.org/v1/graphql'
+
+export const getApolloClient = () => {
```

Comment:
You can do this in a follow-up, but you should leave a TODO for yourself:
This instantiates a new client for every call, so the in-memory cache will never even be used. You need to store this as a global var. You can probably just `export default new ApolloClient...`, instead of having a `getApolloClient` method.